analysis/topeft_run2/fullR2_run.sh

```diff
@@ -1,17 +1,25 @@
 # This script runs the wq run script with all of the settings appropriate for making SR histos for the full R2 analysis
 
 # Name the output
-OUT_NAME="example_name"
+COMMIT="b225975a"
+TAG="run2taus"
 
 # Build the run command for filling SR histos
+TAG="${TAG}_SRs"
+OUT_NAME="${TAG}_${COMMIT}"
 CFGS="../../input_samples/cfgs/mc_signal_samples_NDSkim.cfg,../../input_samples/cfgs/mc_background_samples_NDSkim.cfg,../../input_samples/cfgs/data_samples_NDSkim.cfg"
-OPTIONS="--hist-list ana --skip-cr --do-systs -s 50000 --do-np -o $OUT_NAME" # For analysis
+OPTIONS="--hist-list ana --skip-cr --do-systs -s 50000 --do-np -o $OUT_NAME --tau_h_analysis" # For analysis
```

Comment:
Please remove all the tau specific stuff (including the `COMMIT` an `TAG` above)

Reply:
Done, even though I will propose in my next commit to add a default appendix to `OUT_NAME` to differentiate between SRs and CRs runs. I think it's useful to avoid unintended overwrites of SRs pkl files with CRs pkl files (and vice-versa)